api/tasks.js

```diff
@@ -37,38 +33,26 @@ router.post("/tasks", async (req, res) => {
       });
     }
   }
-  try {
-    const result = await models.Task.create(req.body);
-    return res.status(200).json({ result: result });
-  } catch (e) {
-    return res.status(500).json({ error: e });
-  }
+  const result = await models.Task.create(req.body);
+  return res.status(200).json({ success: result });
 });
 
-router.put("/tasks", async (req, res) => {
-  try {
-    const updatedObj = req.body;
-    const taskId = updatedObj.id;
-    delete updatedObj.id;
-    await models.Task.update({ ...updatedObj }, { where: { id: taskId } });
-    return res.status(200);
-  } catch (e) {
-    return res.status(500).json({ err: e.message });
-  }
+router.put("/:id", async (req, res) => {
+  const updatedObj = req.body;
+  const taskId = updatedObj.id;
+  delete updatedObj.id;
+  await models.Task.update({ ...updatedObj }, { where: { id: taskId } });
+  return res.status(200);
```

Comment:
unless i missed something, is there a reason we don't return the updated task?

Reply:
rolling the get hooks for now, only ones used, going to port the create routes as needed